apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx

```diff
@@ -152,17 +177,86 @@ export const BulkImportUserWizard: FunctionComponent<BulkImportUserInterface> =
     const [ bulkResponseSummary, setBulkResponseSummary ] = useState<BulkResponseSummary>(initialBulkResponseSummary);
     const [ readWriteUserStoresList, setReadWriteUserStoresList ] = useState<DropdownItemProps[]>([]);
     const [ selectedUserStore, setSelectedUserStore ] = useState<string>("");
-    
-    useEffect(() => {
-        setSelectedUserStore(userstore);
-        getUserStoreList();
-    }, [ userstore ]);
     const [ configureMode, setConfigureMode ] = useState<string>(undefined);
     const [ emailData, setEmailData ] = useState<string[]>();
     const [ isEmailDataError, setIsEmailDataError ] = useState<boolean>(false);
     const [ emailDataError, setEmailDataError ] = useState<string>("");
+    const [ roleUserAssociations, setRoleUserAssociations ] = useState<Record<string, RoleUserAssociation>>({});
 
     const optionsArray: string[] = [];
+    
+    const {
+        data: rolesList,
+        isLoading: isRolesListLoading,
+        error: rolesListError
+    } = useRolesList(
+        undefined, undefined, ORG_ROLE_FILTER
+    );
+    
+    /**
+     * Handle if any error occurs while fetching the roles list.
+     */
+    useEffect(() => {
+        if (rolesListError) {
+            dispatch(
+                addAlert({
+                    description: t("console:manage.features.roles.notifications.fetchRoles.genericError.description"),
+                    level: AlertLevels.ERROR,
+                    message: t("console:manage.features.roles.notifications.fetchRoles.genericError.message")
+                })
+            );
+        }
+    }, [ rolesListError ]);
+    
+    /**
+     * Fetch the user roles list.
+     */
+    useEffect(() => {
+        const newRoles: Record<string, RoleUserAssociation> = rolesList?.Resources?.reduce((
+            acc: Record<string, RoleUserAssociation>, role: RolesInterface) => {
+            acc[role?.displayName.toLowerCase()] = {
+                displayName: role?.displayName,
+                id: role?.id,
+                users: []
+            };
+
+            return acc;
```

Comment:
Let's rename this variable to make it clear

Reply:
Fixed